scripts/label_validator.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# standard bash error handling
+set -o nounset  # treat unset variables as an error and exit immediately.
+set -o errexit  # exit immediately when a command fails.
+set -E          # must be set if you want the ERR trap
+set -o pipefail # prevents errors in a pipeline from being masked
+
+PR_ID=$1
+
+kind_labels=()
+
+relase_notes_supported_labels=$(yq eval '.changelog.categories.[].labels' ./.github/release.yml)
+while IFS= read -r label; do
+  clean_label=$(echo "$label" | sed 's/-//g' | sed 's/ //g')
+  if [[ $clean_label == kind* ]]; then
+    kind_labels+=("$clean_label")
+  fi
+done <<< "$relase_notes_supported_labels"
+
+present_labels=$(curl -L \
+                  -H "Accept: application/vnd.github+json" \
+                  -H "X-GitHub-Api-Version: 2022-11-28" \
+                  https://api.github.com/repos/kyma-project/btp-manager/pulls/${PR_ID} | 
+                  jq -r '.labels[] | objects | .name')
```

Comment:
Consider `curl <all the parameters>|grep -cFf <file with labels separated by newlines>` 
It can be done w/o real file, just using file descriptors...